addons/calendar/tests/test_access_rights.py

```diff
@@ -133,3 +133,26 @@ def test_private_attendee(self):
                          "Owner should be able to read the event")
         with self.assertRaises(AccessError):
             self.read_event(self.portal, event, 'location')
+
+    def test_meeting_edit_access_notification_handle_in_odoo(self):
+        # set notifications to "handle in Odoo" in Preferences for john, raoul, and george
+        (self.john | self.raoul | self.george).write({'notification_type': 'inbox'})
+
+        # raoul creates a meeting for john, excluding themselves
+        meeting = self.env['calendar.event'].with_user(self.raoul).create({
+            'name': 'Test Meeting',
+            'start': datetime.now(),
+            'stop': datetime.now() + timedelta(hours=2),
+            'user_id': self.john.id,
+            'partner_ids': [(4, self.raoul.partner_id.id)],
+        })
+
+        # george tries to modify the start date of the meeting to a future date
+        meeting = meeting.with_user(self.george)
+        try:
+            meeting.write({
+                'start': datetime.now() + timedelta(days=2),
+                'stop': datetime.now() + timedelta(days=2, hours=2),
+            })
+        except AccessError:
+            self.fail('AccessError was raised while it should not have.')
```

Comment:
Why do you use try except in a test ?
If the write all fails, there will be a traceback and an exception. of course the dev will know that the `AccessError was raised while it should not have.`

Reply:
You can add a meaningful comment to describe what you are trying to check with this write.
Otherwise you can simply assert the new values start and stop with nice comments when it fails even if the write would fails before.

Reply:
I added the comment.

Reply:
@arj-odoo Hello,
I modified the code. Could you please review it?